app/hypnos/src/backlight.c

```diff
@@ -23,14 +25,18 @@ static bool backlight_enabled = false;
 void backlight_init()
 {
 	backlight_dev = device_get_binding(BACKLIGHT_PORT);
-	gpio_pin_configure(backlight_dev, BACKLIGHT, GPIO_OUTPUT);
+	gpio_pin_configure(backlight_dev, BACKLIGHT_1, GPIO_OUTPUT);
```

Comment:
So issue was floating state for the other backlight gpios ?

Reply:
I suppose so. In any case, the initial state of the leds were different after booting with or without mcuboot.